static/styles.css

```diff
@@ -243,6 +243,69 @@ section {
   }
 }
 
+.search-results-header {
+  margin-bottom: 4px;
+
+  > h1,
+  > h2 {
+    margin: 0;
+  }
+}
```

Comment:
Ugly shit that I have to overwrite some margins to get it decent.  That is the only variation to the pager right side on the homepage.

Reply:
Removed